src/main/resources/templates/ftl-bits/app-js.ftl

```diff
@@ -1,13 +1,21 @@
 <!-- Include local javascript from js/... -->
-<script src="js/app.js"></script>
-<script src="js/controllers.js"></script>
-<script src="js/routes.js"></script>
-<script src="js/services.js"></script>
-<script src="js/mods.js"></script>
-<script src="js/constants.js"></script>
-
+<script type="text/javascript" src="js/app.js"></script>
+<script type="text/javascript" src="js/controllers.js"></script>
+<script type="text/javascript" src="js/routes.js"></script>
+<script type="text/javascript" src="js/services.js"></script>
+<script type="text/javascript" src="js/mods.js"></script>
+<script type="text/javascript" src="js/constants.js"></script>
 <!-- Controllers -->
-<script src="js/new-container-ctrl.js"></script>
-<script src="js/view-containers-ctrl.js"></script>
-<script src="js/upload-mod-ctrl.js"></script>
-<script src="js/manage-mods-ctrl.js"></script>
\ No newline at end of file
+<script type="text/javascript" src="js/new-container-ctrl.js"></script>
+<script type="text/javascript" src="js/view-containers-ctrl.js"></script>
+<script type="text/javascript" src="js/upload-mod-ctrl.js"></script>
+<script type="text/javascript" src="js/manage-mods-ctrl.js"></script>
+<!-- Tui2tone admin v.2 bootstrap -->
```

Comment:
Don't put third party js in `app-js.ftl`. Just files created by us, please. These can go in `ext-js.ftl`